doc/learning-to-rank.md

```diff
@@ -153,6 +153,140 @@ work with this Concourse and SageMaker configuration.
+    ```
+
+3. Add a new training job to the `jobs:` list:
+
+    *If you're not using custom training data, instead use the output
+    of the normal fetch job here.*
+
+    ```yaml
+    - name: integration-AB-SOMETHINGtrain
+      plan:
+        # delete "get: integration-training-data" step
+        - get: search-api-git
+        - task: Train
+          config:
+            <<: *task-config
+            inputs:
+              # delete "name: integration-training-data" input
+              - name: search-api-git
+            outputs:
+              - name: out
+            params:
+              # delete "INPUT_FILE_NAME: training-data" param
+              GOVUK_ENVIRONMENT: integration
+              IMAGE: ((integration-ecr-repository))
+              ROLE_ARN: ((integration-role-arn))
+              OUTPUT_FILE_NAME: AB-SOMETHING-model
+              # add "IMAGE_TAG: <tag name>" param, with the appropriate <tag name>
+              IMAGE_TAG: "<tag name>"
+              # add "SCRIPT_INPUT_DATA: <key name>" param, with the appropriate <key name>
+              SCRIPT_INPUT_DATA: "<key name>"
+            run:
+              path: bash
+              args: ["search-api-git/ltr/concourse/task.sh", "train"]
+          on_failure:
+            <<: *notify-failure
+        - put: integration-AB-SOMETHING-model
+          params:
+            file: out/integration-AB-SOMETHING-model-*.txt
+    ```
+
+4. Add a new deployment job to the `jobs:` list:
+
+    ```yaml
+    - name: integration-AB-SOMETHING-deploy
+      plan:
+        - get: integration-AB-SOMETHING-model
+          passed: [integration-train]
+          trigger: true
+        - get: search-api-git
+        - task: Deploy
+          config:
+            <<: *task-config
+            inputs:
+              - name: search-api-git
+              - name: integration-AB-SOMETHING-model
+            params:
+              GOVUK_ENVIRONMENT: integration
+              ROLE_ARN: ((integration-role-arn))
+              INPUT_FILE_NAME: AB-SOMETHING-model
+              # add "ENDPOINT_NAME: govuk-integration-AB-SOMETHING-search-ltr-endpoint"
+              ENDPOINT_NAME: govuk-integration-AB-SOMETHING-search-ltr-endpoint
+            run:
+              path: bash
+              args: ["search-api-git/ltr/concourse/task.sh", "deploy"]
+          on_failure:
+            <<: *notify-failure
+    ```
+
+**Deploy your pipeline with `fly`,** if you've not logged into
+Concourse before, download `fly` from
+https://cd.gds-reliability.engineering/ (see the list of operating
+systems in the bottom right) and log in:
+
+```bash
+fly login -t cd-govuk-tools -n govuk-tools -c https://cd.gds-reliability.engineering
+```
+
+Then deploy the pipeline:
+
+```bash
+fly set-pipeline -t cd-govuk-tools -p search-learn-to-rank -c ltr/concourse/pipeline.yaml
+```
+
+**Trigger a build of the "train" job,** which will train a new model,
+upload it to S3, and trigger the deploy job.
```

Comment:
What happens to the model once it has been trained and exported to S3? Does it get served automatically? Also, would this be a one-time training, or would a new custom AB test model be trained daily?